tests/integration.test.ts

```diff
@@ -0,0 +1,78 @@
+import { OMGCLI } from "../src/omgcli";
+const config = require("../config.js");
```

Comment:
General comment on this file but wonder whether we should be testing for content of objects and arrays rather than just type conformity?

Reply:
Checking for important attributes now